### DISCUSSION TITLE
Need help with using community preset in runtime

### DISCUSSION THREAD

**Discussion by iarkaroy** [unocss/unocss#2807] [Q&A]
I am looking for the way to use https://github.com/Julien-R44/unocss-preset-forms preset with runtime environment. But I could not find any guide for that in https://unocss.dev/integrations/runtime. How should I use the preset with runtime?

Any guidance in the right direction will be much appreciated.

Thanks

**Comment by maheshveact** [comment]
Hi @iarkaroy ,

did you found the solution for this use case?
